attic/rfc_metrics.md

```diff
@@ -20,13 +20,23 @@ I propose when applying a metric method, what occurs should be ...
 
 Example:
 ```r
-#The true label of the observations
-truth = as.factor(c(1,1,1,0,0,1,1,0,0,0))
-
-#The predicted labels of the observations
-response = as.factor(c(1,1,0,0,1,1,0,0,0,1))
-
-fn(truth, response, positive = "1")
+#Initialize the data task and learner task
```

Comment:
Feel free to add a few more sentences to the RFC.

Reply:
Describe in detail what the current open questions are.
This is the basis for discussions with the other Mentors